p4-14/v1.0.5/tex/p4.pt

```diff
@@ -1711,7 +1728,7 @@ Drop a packet (in the egress pipeline). \\ \hline
 \texttt{no_op} &
 Placeholder action with no effect. \\ \hline
 \texttt{push} &
-Push all header instances in an array down and add a new header at the top. \\ \hline
+Push all header instances in an array down and add a new invalid header at the top. \\ \hline
```

Comment:
With recent change to preserve P4_14 push behavior of adding valid headers, this addition of "invalid" should be removed.

Reply:
Good catch. Fix.